sdk/messaging/eventgrid/aznamespaces/models.go

```diff
@@ -53,6 +49,10 @@ type FailedLockToken struct {
 	LockToken *string
 }
 
+// publishResult - The result of the Publish operation.
+type publishResult struct {
```

Comment:
Is this something that's being generated?

Reply:
Yes.

Reply:
Mostly I was wondering if this was a base type that the emitter should have pruned. But that doesn't appear to be the case. It's harmless enough especially as it's not exported, but it does seem a bit weird. Might be worth following up with the event grid folks on why it's authored like this and if there's a better pattern.

Reply:
So I think I can just not use it at all (it is embedded in public responses, which is a big no-no). 

This all hangs on what they said in the last arch-board meeting about publish operations _never_ returning a result. So my instinct is to remove this, and change the signature of Send and SendEvents to only return only an 'error'. What do you think?

Reply:
I see. I think we should continue to return the response envelopes, just omit the `publishResult` type from them. This is consistent with our SDK patterns and is extensible in a non-breaking way.

Reply:
Brought it all back, we'll leave it as generated (except publishResult will be exported)